archive/tool/get_comments.sh

```
@@ -0,0 +1,45 @@
+          path
+          line
+          comments(first:10){
+            nodes{
+              body
+            }
+          }
+        }
+      }
+    }
+  }
+}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)